packages/engine/Source/Core/ITwinPlatform.js

```diff
@@ -0,0 +1,155 @@
+ *
+ * @type {string|undefined}
+ */
+ITwinPlatform.defaultAccessToken = undefined;
+
+/**
+ * Gets or sets the default iTwin API endpoint.
+ *
+ * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
+ *
+ * @type {string|Resource}
+ * @default https://api.bentley.com
+ */
+ITwinPlatform.apiEndpoint = new Resource({
+  url: "https://api.bentley.com",
+});
+
+/**
+ * @typedef {Object} ExportRequest
+ * @property {string} iModelId
+ * @property {string} changesetId
+ * @property {ITwinPlatform.ExportType} exportType Type of the export. CesiumJS only supports the 3DTILES type
+ */
+
+/**
+ * @typedef {Object} Link
+ * @property {string} href
+ */
+
+/**
+ * @typedef {Object} ExportRepresentation
+ * The export objects from get-exports when using return=representation
+ * @property {string} id Export id
+ * @property {string} displayName Name of the iModel
+ * @property {ITwinPlatform.ExportStatus} status Status of this export
+ * @property {string} lastModified
+ * @property {ExportRequest} request Object containing info about the export itself
+ * @property {{mesh: Link}} _links Object containing relevant links. For Exports this includes the access url for the mesh itself
+ */
+
+/**
+ * @typedef {Object} GetExportsResponse
+ * @property {ExportRepresentation[]} exports The list of exports for the current page
+ * @property {{self: Link, next: Link | undefined, prev: Link | undefined}} _links Pagination links
+ */
+
+/**
+ * Get the list of exports for the specified iModel at it's most current version. This will only return exports with {@link ITwinPlatform.ExportType} of <code>3DTILES</code>.
+ *
+ * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
+ * @private
+ *
+ * @param {string} iModelId iModel id
+ *
+ * @throws {RuntimeError} Unauthorized, bad token, wrong scopes or headers bad.
+ * @throws {RuntimeError} Not allowed, forbidden
+ * @throws {RuntimeError} Unprocessable Entity
+ * @throws {RuntimeError} Too many requests
+ * @throws {RuntimeError} Unknown request failure
+ * @returns {Promise<GetExportsResponse>}
+ */
+ITwinPlatform.getExports = async function (iModelId) {
+  //>>includeStart('debug', pragmas.debug);
+  Check.typeOf.string("iModelId", iModelId);
+  if (!defined(ITwinPlatform.defaultAccessToken)) {
+    throw new DeveloperError("Must set ITwin.defaultAccessToken first");
+  }
+  //>>includeEnd('debug')
+
+  const resource = new Resource({
+    url: `${ITwinPlatform.apiEndpoint}mesh-export`,
+    headers: {
+      Authorization: `Bearer ${ITwinPlatform.defaultAccessToken}`,
+      Accept: "application/vnd.bentley.itwin-platform.v1+json",
+      Prefer: "return=representation",
+    },
+    queryParameters: {
+      iModelId: iModelId,
+      exportType: ITwinPlatform.ExportType["3DTILES"],
+      // TODO: If we're only requesting the top 1 is there a chance it's `Invalid` instead of `Complete`
```

Comment:
Address or remove this TODO. 

I think we may have discussed offline that if the state is `Invalid`, then we do the simplest thing: Either return the response as-is or throw an error. I think I'm leaning towards to former.

Reply:
I think this is a question for @danieliborra.

When I was exploring the API thus far (and earlier using start-export multiple times) there was a chance there would be multiple exports for the same version of an iModel but some were `Invalid` and some were `Complete`.

I don't know how it will be handled in the auto-generation.
1. If the export fails for the type we need will the export get regenerated?
2. Will the `Invalid` export still be in the list?

Depending on the answer here I can update this.

My concern is that this API request returns a list and I don't see any way to filter the API request based on it's status. If `Invalid` exports don't get deleted then there's a chance the full page of requests we get returned would all be invalid (in this case only 1 but same concern for larger pages). If that's the case even if we do return that result to the user they can't do anything about it because we removed the ability to start exports or load specific export ids.

Reply:
You're completely right. I'd suggest the following:
- Get the latest 5 exports => $top=5
- Get the first one with status == "Complete"

If you don't find any, wait 15 seconds and try again.

Reply:
Discussed with @danieliborra offline and found the piece of the puzzle I was missing.

- The auto generation of exports automatically deletes the 6th export if it exists

Basically meaning there _should_ never be more than 5 exports. If they're all invalid then there are bigger problems with that iModel. With that info I'm perfectly fine to only request the `top=5` results and have updated the code to account for this.